spec/game_factory_spec.rb

```diff
@@ -26,4 +26,37 @@
       expect(@game_factory.create_games.all? {|game| game.class == Game}).to be true
     end
   end
+
+  describe '#total_score' do
+    it 'should return array of integers that is the sum of away_goals and home_goals' do
+      @game_factory.create_games
+
+      expect(@game_factory.total_score).to be_a(Array)
+      expect(@game_factory.total_score).to eq([5, 5, 3, 5, 4, 3, 5, 3, 1, 3, 3, 4, 2, 3, 5, 3, 4, 4, 5, 5])
+
+    end
+  end
+  #we only have post_season for our fixture file here. Should we include regular_season games as well?
```

Comment:
Good call out. We can have Laura add some more to our fixture files for added variety and robust testing!